server/lobbyconnection.py

```diff
@@ -551,15 +551,16 @@ async def command_hello(self, message):
                 )
             )
 
-            conforms_policy = await self.check_policy_conformity(
-                player_id, message['unique_id'], self.session,
-                ignore_result=(
-                    steamid is not None or
-                    self.player_service.is_uniqueid_exempt(player_id)
+            if config.USE_POLICY_SERVER:
+                conforms_policy = await self.check_policy_conformity(
```

Comment:
Could also do the check inside `check_policy_conformity` as
```python
if not config.USE_POLICY_SERVER:
    return True
```
to save one indentation level.
Which one is easier to read is up to personal taste I think :D

Reply:
True